rollup.config.js

```diff
@@ -56,28 +56,21 @@ const config = [
     plugins: [
       ...plugins,
       babel({
-        // The 'external-helpers' Babel plugin allows Rollup to include every
-        // used babel helper just once per bundle, rather than including them in
-        // every module that uses them (which is the default behaviour).
-        plugins: ['external-helpers'],
```

Comment:
Removed because warning:

>(!) Plugin babel: Using "external-helpers" plugin with rollup-plugin-babel is deprecated, as it now automatically deduplicates your Babel helpers.